locales/de.json

```diff
@@ -0,0 +1,341 @@
+        "accept": "Neues Projekt",
+        "header": "Moment!",
+        "message": "Haben Sie Ihr aktuelles Projekt gespeichert? Wenn Sie fortfahren, werden alle ungesicherten Änderungen verworfen.",
+        "reject": "Abbrechen"
+      },
+      "new_update_available": {
+        "header": "Aktualisierung verfügbar",
+        "later": "Später",
+        "message": "Die BULB-Version {version} ist nun verfügbar. Möchten Sie die Seite aktualisieren, um auf die neue Version zu wechseln? Dadurch gehen keine Daten verloren.",
+        "update": "Aktualisieren"
+      },
+      "ornament_editor": {
+        "airport": "Flughafen",
+        "display_ornament": "Hinweis hinzufügen",
+        "header": "Optionen",
+        "name": "Name",
+        "no_ornament_available": "Kein Hinweis verfügbar",
+        "ornament": {
+          "airport": "Flughafen",
+          "airport_name": "Name des Flughafens",
+          "text": "Text"
+        },
+        "position": {
+          "below": "unten",
+          "right": "rechts",
+          "title": "Position"
+        },
+        "text": "Text"
+      },
+      "outdated_project": {
+        "accept": "Neues Projekt",
+        "header": "Nicht unterstützte Projektversion",
+        "message": "Das Projekt ist in einer nicht mehr unterstützten Version gesichert. Wenn Sie dennoch fortfahren, können Probleme auftauchen. Es wird empfohlen, ein neues Projekt oder ein Projekt in unterstützter Version zu öffnen.",
+        "reject": "Trotzdem fortfahren"
+      },
+      "parallel_branches_properties": {
+        "alignment": {
+          "fluid": "passend",
+          "left": "links",
+          "right": "rechts",
+          "title": "Orientierung"
+        },
+        "header": "Paralleler Abschnitt: Optionen",
+        "section_1_level": "Ebene des Abschnitt 1",
+        "section_2_level": "Ebene des Abschnitt 2"
+      },
+      "save_project": {
+        "bundle_custom_indices": "benutzerdefinierte Liniensignale mitspeichern",
+        "cancel": "Abbrechen",
+        "header": "Projekt speichern",
+        "project_name": "Name des Projekts",
+        "save": "Speichern"
+      },
+      "spacer_properties": {
+        "header": "Flexibler Abschnitt: Optionen",
+        "size": "Länge",
+        "spacer_export_notice": "Anmerkung: Die Hilfspfeile werden beim Export nicht angezeigt."
+      },
+      "stop_properties": {
+        "city_name": "Ort",
+        "closed": "Aufgelassen",
+        "header": "Haltestelle: Optionen",
+        "interest_point": "relevanter Ort",
+        "stop_name": "Name",
+        "subtitle": "Untertitel",
+        "terminus": "Endstation"
+      },
+      "troubleshooting": {
+        "github": "erstellen Sie ein Issue auf GitHub",
+        "header": "Problembehandlung",
+        "notice": "Folgende Informationen können bei Fehlern in BULB zur Problembehandlung beitragen. Sollten Sie einen Fehler entdecken, {github} oder {twitter}, en précisant les informations ci-dessous.",
```

Comment:
Meh, just spotted some forgotten french here... If you could just give me the missing translation

Reply:
Oh sorry, I missed that 😣 That should be:`       "notice": "Folgende Informationen können bei Fehlern in BULB zur Problembehandlung beitragen. Sollten Sie einen Fehler entdecken, {github} oder {twitter}, und übermitteln Sie mit der Problembeschreibung die untenstehenden Informationen.",`

Reply:
Thanks!